packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -6,36 +6,18 @@ pm2 list
 echo "Deleting..."
 pm2 delete all
 
-# Set the path of environment variables from parameter store
-if [[ $BRANCH == "master" ]]; then
-    ENVIRONMENT="production"
-else
-    # Any other branch uses the default dev environment variables
-    ENVIRONMENT="dev"
```

Comment:
We still need this, no? Feature branches should pull down the lastpass note for [Dev] right?

Reply:
Yeah but that's now built into `downloadEnvironmentVariables.sh` 🎉